flow/libs/intl.js.flow

```diff
@@ -1,5 +1,4 @@
-// This is only a partial definition of
-// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl
+// This is only a partial definition of Intl (https://mzl.la/1dsIHLk MDN)
 
 declare class Intl$NumberFormat {
   constructor(locales: string | Array<string>, options?: Object): void;
```

Comment:
I just noticed that `format(number: Number)` should be `format(number: number)` -- [the lower case one is what we want](https://flow.org/en/docs/types/primitives/#toc-numbers). I always forget about the difference. Can you add that change? It doesn't help with the bug we're hitting but it will prevent other errors.